README.md

```diff
@@ -114,7 +114,7 @@ Because it is more convenient, faster, and easier to use.
 ### Styled Text
 This module enables you to style the terminal text.
 
-Good documentation can be found at the following places: [docs](https://docs.rs/crossterm_style/), [book](https://timonpost.github.io/crossterm/docs/styling.html), [examples](https://github.com/TimonPost/crossterm/tree/master/examples/key_events.rs)
+Good documentation can be found at the following places: [docs](https://docs.rs/crossterm_style/), [book](https://timonpost.github.io/crossterm/docs/styling.html), [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples/key_events.rs)
```

Comment:
I'd suggest to generate pages, setup them now and change these links as well. Pages are the only thing that are not redirected.